### ISSUE TITLE
Color code in "tag liés"

### ISSUE THREAD

**Issue by ericbl** [Piwigo/Piwigo-Colored-Tags#2]
Je viens de faire la MAJ 2.8.3
Dans le menu des tags liés, le code couleur en hexa est affiché et tout est en noir, au lieu d avoir les tags colorés. Par ex ds un de mes albums. Le menu tag liés affiche #7dff42;">by night #85d4ff;">Austria #ffbe80;">friend
Update nécessaire?

**Comment by mistic100** [comment]
Aucun problème chez moi, à voir avec votre thème qui doit echapper le HTML dans les tags.

**Comment by ericbl** [comment]
cela c est du support express! Chapeau!
J utilise le theme 'dark' par défaut. Une idée de ce qui pourrait être mal configuré chez moi?
Ex d un album sur ma galerie: http://photo.eric-creation.com/index.php?/category/358

**Comment by mistic100** [comment]
Non. C'est certainement une incompatibilité, mais je ne peux pas vous dire avec quoi.

Et je ne développe plsu en PHP...

**Comment by bigsfr** [comment]
J'avais le meme pb.
J'ai corrigé en changeant dans include/events_public.inc.php
`    $ret = '<span style="color:' . $color . ';">' . $tag_name . '</span>';`
par 
`    $ret = ':<span style="color:' . $color . ';">' . $tag_name . '</span>';`

Apparement l'idée c'est qu'il fait sauter le premier ":" et tout ce qui il y a avant.
pas cherché le pourquoi du comment par contre.

**Comment by ericbl** [comment]
merci pour l astuce, bigsfr, 
j ai donc tenté d'éditer de même le fichier plugins/typetags/include/events_public.inc.php
Malheureusement cela reste sans effet chez moi, sur ma combinaison de thème et plugins.
